examples/step-12/step-12.cc

```diff
@@ -159,20 +227,19 @@ namespace Step12
   private:
     void setup_system();
     void assemble_system();
-    void solve(Vector<double> &solution);
+    void solve();
     void refine_grid();
     void output_results(const unsigned int cycle) const;
 
     Triangulation<dim>   triangulation;
     const MappingQ1<dim> mapping;
 
-    // Furthermore we want to use DG elements of degree 1 (but this is only
-    // specified in the constructor). If you want to use a DG method of a
-    // different degree the whole program stays the same, only replace 1 in
-    // the constructor by the desired polynomial degree.
+    // Furthermore we want to use DG elements.
     FE_DGQ<dim>     fe;
     DoFHandler<dim> dof_handler;
 
+    AffineConstraints<> constraints;
```

Comment:
Couldn't we just as well create this object wherever we need it?

Reply:
done.